feature/selector.py

```diff
@@ -5,18 +5,20 @@
 
 """
 :Author: FMR LLC
-:Version: 1.0.0 of August 10, 2020
+:Version: 1.1.0 of August 10, 2020
```

Comment:
Update date too 😊

Reply:
Oh yes! I wasn't sure if 1.1.0 would be the current version. Should I also update `_version.py` to 1.1.0 then?

Reply:
Yes please and add remember to add entry to `CHANGELOG.txt` too.